src/panels/config/scene/ha-scene-editor.ts

```diff
@@ -815,6 +814,25 @@ export class HaSceneEditor extends SubscribeMixin(
     );
   }
 
+  private _calculateMetaData(): SceneMetaData {
+    const output: SceneMetaData = {};
+    const filteredEntityReg = this._entityRegistryEntries.filter((entityReg) =>
+      this._entities.includes(entityReg.entity_id)
+    );
+
+    for (const entityReg of filteredEntityReg) {
+      const entityState = this._getCurrentState(entityReg.entity_id);
+      if (entityState) {
+        output[entityReg.entity_id] = {
+          entity_only: !!(
+            entityReg.device_id && !this._devices.includes(entityReg.device_id)
+          ),
+        };
+      }
```

Comment:
You can simplify this. We only need to set this when it's `true`
```suggestion
      if (entityState && entityReg.device_id && !this._devices.includes(entityReg.device_id) {
        output[entityReg.entity_id] = {
          entity_only: true,
        };
      }
```

Reply:
That's correct for the current implementation, but isn't it clearer if the flag is always set?